pkgs/by-name/ld/ld64/gen_compile_stubs.py

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env python
```

Comment:
If this is bikeshedded at all, it should probably stay Python so it can spit out a more efficiently‐compiling string literal rather than a bunch of integer literal nodes, as I discussed on Matrix. (Although maybe there’s a nice way to C‐escape a byte string in shell, I dunno. I’d rather see a Python script than a Bash one any day, personally.)

Reply:
yeah, i was thinking if it reduces the number of times python needs to be built in the bootstrap. if it is already built it doesn't really matter.

Reply:
I was going to say the same thing. If I made any change, I’d rather make it produce a string literal than rewrite in bash. I don’t particularly like scripting in bash, and this seems less readable than the Python. Python is also already a Darwin bootstrap requirement, so requiring it for ld64 does not add another dependency to the bootstrap.

Regarding csh, I don’t think it matters. I don’t know what anyone even does with the symbol, but ld64 embeds the string in the compiled binary, so I’ve included it to match what upstream does.

Reply:
> yeah, i was thinking if it reduces the number of times python needs to be built in the bootstrap. if it is already built it doesn't really matter.

Python is built only three times (minimal once, full twice). Ideally, Python would be built only twice (minimal once and fully once), but eliminating the extra Pythons is difficult.

Reply:
Also note that I’m probably not going to change it to produce the string literal because the script isn’t big enough that it really matters for compile performance. Eventually, it might be nice to switch to C23 `#embed`, but that just landed a few weeks ago in clang. It probably won’t be available for Darwin to use until next year’s clang 20 bump.